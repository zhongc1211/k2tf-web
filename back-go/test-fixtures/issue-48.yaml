apiVersion: v1
kind: ReplicationController
metadata:
  name: echoheaders
  namespace: platform
spec:
  replicas: 1
  template:
    metadata:
      labels:
        app: echoheaders
    spec:
      selector:
        app: echoheaders
      containers:
        - name: echoheaders
          image: k8s.gcr.io/echoserver:1.10
          ports:
            - containerPort: 8080
