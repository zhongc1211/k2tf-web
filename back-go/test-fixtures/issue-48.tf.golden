resource "kubernetes_replication_controller" "echoheaders" {
  metadata {
    name      = "echoheaders"
    namespace = "platform"
  }
  spec {
    replicas = 1
    template {
      metadata {
        labels = { app = "echoheaders" }
      }
      spec {
        container {
          name  = "echoheaders"
          image = "k8s.gcr.io/echoserver:1.10"
          port {
            container_port = 8080
          }
        }
      }
    }
  }
}
